### DISCUSSION TITLE
essay/2023/08/25/befriending-someone-you-wanted-to-kill

### DISCUSSION THREAD

**Discussion by giscus** [BenjaminKim/benjaminkim.github.io#121] [Comments]
# essay/2023/08/25/befriending-someone-you-wanted-to-kill

책 ‘마음 가는 대로 해라’ 중

https://jeho.page/essay/2023/08/25/befriending-someone-you-wanted-to-kill.html

**Comment by roeniss** [comment]
관점을 뒤짚어버리는 의견을 준 친구분도, 그 의견에 "그래 한 번 해보자" 하며 실천한 벤자민도 대단하네요. 여러가지 생각이 드는 밤입니다.

**Reply by BenjaminKim** [reply]
하하 그렇죠? 요즘엔 저런 힘든 일이 없어서 다행.. 그리고 좋은 친구가 옆에 있어서 다행입니다.
